acme/dns_challenge.go

```diff
@@ -44,6 +44,10 @@ func (s *dnsChallenge) Solve(chlng challenge, domain string) error {
 		return err
 	}
 
+	// sleep a little bit to let the record propigate.
```

Comment:
This is the only other change I made besides merging. Temporary, but it makes things work until querying is implemented.